examples/arm/run.sh

```diff
@@ -28,7 +28,7 @@ setup_path_script=${root_dir}/setup_path.sh
 
 # Executorch
 et_root_dir=$(cd ${script_dir}/../.. && pwd)
-et_build_dir=${root_dir}/executorch-cmake-out
+et_build_dir=${root_dir}/cmake-out
```

Comment:
what's the rationale behind this change? can't we install in the `${et_build_dir}`?

Reply:
I got trouble with finding package